### ISSUE TITLE
Alternate email or sub-account on customer registration [Feature Request]

### ISSUE THREAD

**Issue by heberq1978** [FOSSBilling/FOSSBilling#715]
I'm registering some clients and what I'm missing is an alternative email or a sub account to register more than one person, such as:
You can register a customer as the account holder and another as the person responsible for the payment, which in this case would be a subaccount.
The advantage of a sub-account is that not just one person will have access to that customer's dashboard, where more than one person can manage the same account.
Another advantage is that the account owner will be able to register a developer within his account, so that the developer can talk directly to us, without the need for the developer to have full access to the customer's account.
He will only have access to what the account owner authorizes him to do.

**Comment by John-S4** [comment]
When I read the term sub-accounts I imagine something really complex, but basically what we are talking about (at the simplest level) is just being able to share access to an account between more than one user, is that right?

**Comment by jaapmarcus** [comment]
Simplest method would be add a new column and use the UserID as the owner. Question only how far do we want to implement it if it is a full option to set access so that a sub account can't create/cancel new orders that it will become way to complicated...

**Comment by heberq1978** [comment]
> Quando leio o termo subcontas imagino algo realmente complexo, mas basicamente o que estamos falando (no nível mais simples) é apenas poder compartilhar o acesso a uma conta entre mais de um usuário, está certo?

**Comment by heberq1978** [comment]
> When I read the term sub-accounts I imagine something really complex, but basically what we are talking about (at the simplest level) is just being able to share access to an account between more than one user, is that right?

Yes, that would be more or less, but in a system that I have, the customer can register to buy the things he needs, and then he can register other people for!

**Comment by jaapmarcus** [comment]
Why did you close the issue?

**Comment by heberq1978** [comment]
> Why did you close the issue?

Thanks for talking, I ended up closing it by accident!

**Comment by evrifaessa** [comment]
This would be a nice feature. Though there must be some kind of a permission system like Jaap said.
